components/table.stories.tsx

```diff
@@ -376,12 +376,21 @@ export const Appendable: Story = {
       >
         {rows.map((rowValue, rowIndex) => (
           <Table.Row
+            onClick={() =>
```

Comment:
let's add capability to check the row (when it has a checkbox):

```ts
onClick={({setIsChecked}) => {
  /* make the row checked; but if the row doesn't have a 
     checkbox, calling setIsChecked will not do anything, 
     please ensure this behavior in a test */
  setIsChecked(true);
})
```

this way, we don't assume that clicking the row will trigger checking -- that'd be user-defined behavior.

please make the example "Table / Sortable With Pagination" modified such that clicking the row will also set the checkbox of the row to `checked`.